tox.ini

```diff
@@ -12,8 +12,9 @@ skip_install = true
 deps =
     -r requirements/test.txt
 commands =
-    ruff check . --exit-zero {posargs}
     ruff check . --select E --select F {posargs}
```

Comment:
Also... sorry not to have stated this initially, I expected some discussion around this, but isn't the use of only E and F here misleading since the ``pyproject.toml`` config selects all?

Reply:
this is primarily to transition. Then we can backport to existing projects while letting them still build without intervention. This repo even has a number of violations with new rules.

I was aiming to treat these as useful hints that show up in IDE and that we iterate toward. For example there is a set of pandas rules that seem pretty cool, but I don't know about a hard enforcement on those yet, especially in existing repos.

I would encourage any new work we do to eliminate the selections (as stated in comment), and if by so doing we find annoying ones rules pop up, we come back here and update the ignores.